test/stats.spec.ts

```diff
@@ -13,4 +13,76 @@ describe("when calculating stats", () => {
     expect(p2Success).toBe(5);
     expect(p2Fail).toBe(4);
   });
+  describe("When Handling Command Grabs", () => {
+    it("accounts for Bowser's Command Grab", () => {
+      const game = new SlippiGame("slp/consistencyTest/BowsVDK-SB-63.slp");
+      const stats = game.getStats();
+      const bowser = stats.overall[0]
+      let totalDamageBowserDealt = 0
+      stats.conversions.map( conversion => {
```

Comment:
```suggestion
      stats.conversions.forEach( conversion => {
```
Minor pet-peeve, can you replace all the `map` with `forEach`? In these cases we don't really care about the actual mapping of one array into another array, rather we just care about iterating through each value.

Reply:
You'd need to repeat this change for all the other `.map` instances btw. Not just this one instance.